### PR TITLE
Improve phan config

### PR DESCRIPTION
**Pull request by MorrisJobke**
* exclude routes
* ignore some more templates of apps

Part of #8375 to get it in in smaller steps

### CODE REVIEW

File: build/.phan/config.php [hunk 1]
```diff
@@ -40,56 +47,103 @@
...
-	"exclude_analysis_directory_list" => [
+	'exclude_analysis_directory_list' => [
 		'3rdparty',
-		'lib/composer',
 		'apps/admin_audit/composer',
 		'apps/admin_audit/tests',
 		'apps/comments/composer',
 		'apps/comments/tests',
 		'apps/dav/composer',
+		'apps/dav/templates',
```
Comment (nickvergessen) [inline build/.phan/config.php]: Hmm this caught php errors in templates, any reason to exclude them?

Comment (MorrisJobke) [reply build/.phan/config.php]: Because there are many undefined variables from phan‘s point of view :/

**Review by rullzer — APPROVED** [review]

File: build/.phan/config.php (hunk 1 above)
Comment (MorrisJobke) [reply build/.phan/config.php]: Let's drop this for now and get the remaining stuff in.

**Comment by MorrisJobke** [comment]
@nickvergessen Please re-review ;)

**Review by nickvergessen — APPROVED** [review]

**Pull request merged**